github2fedmsg/views/auth.py

```diff
@@ -15,7 +15,7 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 from pyramid.view import view_config
-from pyramid.httpexceptions import HTTPFound
+from pyramid.httpexceptions import HTTPFound, HTTPForbidden
```

Comment:
I don't see this used in any other change in this file.

Reply:
Yeah it was a missing import that I fixes in drive-by mode ;-)